### DISCUSSION TITLE
Security Onion Console sign-in authentication logs

### DISCUSSION THREAD

**Discussion by northernrevival** [Security-Onion-Solutions/securityonion#6691] [Unsupported Versions]
Hello world,

We have recently discovered something a little curious about an analyst's SOC account. We have a mild suspicion that someone else may have logged into and been active on their SOC account. We have browsed google, the SO docs, and this discussion feed and can't seem to find the answer to:

Where are the Security Onion Console sign-in logs?

We would like to check if the account was accessed outside of the periods in which said analyst was active.

We are running SO version 2.3.90 on CentOS 7, though not entirely sure if that is relevant in this case.

Any help or suggestions will be greatly appreciated.

Kind regards,
NR

**Comment by dougburks** [comment]
Have you checked the Kratos logs in `/opt/so/log/kratos/`?

**Reply by northernrevival** [reply]
Hi @dougburks ,

Many thanks for your suggestion, I've unzipped the kratos log from the 24/12/2021 and tried to:

$ cat kratos-20211224.log | grep <username/email>

but no luck. Having a scroll through the log it looks like connection requests rather than console auth. I'll keep digging around the logs in this area, though!

**Reply by dougburks** [reply]
Try searching for auth logs as follows:
```
sudo zgrep "Identity authenticated successfully and was issued an Ory Kratos Session Cookie" /opt/so/log/kratos/*
```

Once you see the auth logs, you will notice that Kratos logs using `identity_id`. You can find your desired `identity_id` as follows, replacing USERNAME@DOMAIN.COM with your desired SOC username:
```
echo "select * from identities;" | sudo sqlite3 /opt/so/conf/kratos/db/db.sqlite |grep USERNAME@DOMAIN.COM | cut -d\| -f1
```
